corehq/apps/geospatial/models.py

```diff
@@ -128,3 +129,19 @@ class GeoPolygon(models.Model):
     name = models.CharField(max_length=256)
     geo_json = models.JSONField(default=dict)
     domain = models.CharField(max_length=256, db_index=True)
+
+
+class GeoConfig(models.Model):
+
+    CUSTOM_USER_PROPERTY = 'custom_user_property'
+    ASSIGNED_LOCATION = 'assigned_location'
+
+    VALID_LOCATION_SOURCES = [
+        CUSTOM_USER_PROPERTY,
+        ASSIGNED_LOCATION,
+    ]
+
+    domain = models.CharField(max_length=256)
```

Comment:
If you only want one per domain, you could set `domain` as the primary key. Alternatively, since you're going to fetch instances by domain, you should index it.

Reply:
Ah yes, good point.